docs/semgrep-code/semgrep-assistant-code.md

```diff
@@ -192,11 +192,11 @@ The following are recommendations users can receive from Semgrep Assistant.
 
 ### Analyze if a finding is a true or false positive
 
-Semgrep Assistant can analyze if your finding is a true or false positive. The accuracy of its recommendations is roughly 60% and varies based on the language and framework you are using. This is its default use case.
+Semgrep Assistant can analyze if your finding is a true or false positive. The accuracy of its recommendations is roughly 60% and varies based on the language and framework you are using. This is its default use case. This feature is active in both PR/MR comments and Slack notifications.
```

Comment:
```suggestion
Semgrep Assistant can analyze if your finding is a true or false positive. The accuracy of its recommendations is roughly 60% and varies based on the language and framework you are using. This is its default use case. This feature is active in both PR and MR comments and Slack notifications.
```

Reply:
@khorne3 Thanks, I wasn't sure how best to refer to the comments here - I know we don't love the PR/MR notation. This change does feel a tad weird to me because without any further punctuation, this can sound like "in [PR] and [MR comments] and [Slack notifications]". I don't think it's a super likely interpretation so I may just go ahead and commit the suggestion, but if you have thoughts on improvement I'd be happy to do a quick followup.

Reply:
@armchairlinguist Yeah, I can see that -- I've taken to writing "pull request and merge request comments" which I understand is a bit wordy but a bit clearer (maybe? Ha.)